### DISCUSSION TITLE
How much time need to get copilot access after being approve and verified student?

### DISCUSSION THREAD

**Discussion by NafiRahim** [community/community#59334] [GitHub Education] [answered]
### Hi everyone,

How much time do I need to get the benefits of copilot?

(I receive this on benefit page)
Congrats! Your request for [@NafiRahim](https://github.com/NafiRahim) was approved and your benefits will be available soon.

**Comment by vincent-mailhot** [comment]
I think it's about 4 days or so. For me it's been 3 days since I register as a student and my demand was aproved so I'm waiting.

**Reply by NafiRahim** [reply, marked as answer]
> I think it's about 4 days or so. For me it's been 3 days since I register as a student and my demand was aproved so I'm waiting.

Yes. For me it took 3 days and few hours.
